cmd/hlb/command/run.go

```diff
@@ -327,7 +327,7 @@ func Run(ctx context.Context, cln *client.Client, uri string, info RunInfo) (err
 	return err
 }
 
-func displayError(ctx context.Context, w io.Writer, err error, printBacktrace bool) (numErrs int) {
+func DisplayError(ctx context.Context, w io.Writer, err error, printBacktrace bool) (numErrs int) {
```

Comment:
Exporting it here so the `shell-on-error` ControlDebugger implementation can use it.